web_tree_many2one_clickable/readme/USAGE.rst

```diff
@@ -0,0 +1,2 @@
+Put the mouse pointer over a many2one cell and click the button.
+.. image:: ./static/img/clickable.gif
```

Comment:
Let an empty line in between for proper RST formatting and use `..` instead of `.` for being working in this file and the general one.

Reply:
Done, thanks!